spras/util.py

```diff
@@ -76,3 +76,14 @@ def raw_pathway_df(raw_pathway_file: str, sep: str = '\t', header: int = None) -
         df = pd.DataFrame(columns=['Node1', 'Node2', 'Rank', 'Direction'])
 
     return df
+
+def duplicate_edges(df: pd.DataFrame):
```

Comment:
I like this new design of checking for duplicate edges in parse outputs. However, see the comment on the Omics Integrator test case about whether we need to have more complex duplicate checking logic.

Reply:
I updated it to remove duplicate rows now; treats undirected and directed edges coming between 2 nodes as separate edges that the original algorithm should have dealt with.